### DISCUSSION TITLE
Mobile Model-Viewer and Scene Viewer performance in 3D model

### DISCUSSION THREAD

**Discussion by GabrielArakaki** [google/model-viewer#2232] [Q&A]
### Description

Hi! Glitch below for testing!

We have this 3D model that is not performing well in model-viewer mobile and Scene Viewer.
In my device Moto X4, the 3D doesn't load and there is a crash loading model-viewer on the web.

Loading the scene viewer directly from the GLB we also have a lagged experience moving the object which could be replicated on https://arvr.google.com/scene-viewer-preview, with an experience way worse than model-viewer's on desktop.

We are currently basing our 3d modeling pipeline on the scene viewer [recommended limits](https://developers.google.com/ar/develop/java/scene-viewer), but in this specific 3D model we have respected all limits in its lower level. Could you help me with any other metric/limit that we could monitor on our 3D modeling pipeline to prevent those performance issues on the Viewers? Tks

Mobile crash:
```
WebGL: CONTEXT_LOST_WEBGL: loseContext: context lost
THREE.WebGLRenderer: Context Lost. 
```

#### Live Demo

https://glitch.com/~pentagonal-zippy-hexagon

### Version

- model-viewer: vX.X.X

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Yeah, it doesn't surprise me that this model has poor performance. Representing fuzziness with lots of triangle shards is not something shaders do efficiently. Is this also using alpha-channel cutouts? I'm not sure if that would make it better or worse, but you could try. You'll have to do some experimentation, but hairy things are not easy to represent in 3D without very specialized shaders.
